process.c

```diff
@@ -540,18 +541,15 @@ get_ppid(void)
 
 /*
  *  call-seq:
- *     Process.ppid   -> integer
+ *    Process.ppid -> integer
  *
- *  Returns the process id of the parent of this process. Returns
- *  untrustworthy value on Win32/64. Not available on all platforms.
+ *  Returns the process ID of the parent of the current process:
  *
- *     puts "I am #{Process.pid}"
- *     Process.fork { puts "Dad is #{Process.ppid}" }
+ *    Process.ppid # => 16112
```

Comment:
We should use a similar example as the previous one and show that the `ppid` does not change across forking.

Reply:
Similar to which example, please?

Reply:
The example that was removed. So something like:

```ruby
puts "PID is #{Process.pid}"
fork { puts "Parent PID is #{Process.ppid}" }
```

Reply:
Restored.